mdakits/packing_defect/metadata.yaml

```
@@ -0,0 +1,59 @@
+# MDAKit metadata for packing_defect
+# See https://mdakits.mdanalysis.org/registering_a_kit/metadata-yaml.html
+
+project_name: packing_defect
+
+authors:
+  - R. Jay Braun Jr.
+
+maintainers:
+  - jaybraunjr
+
+description: >-
+  Analyze membrane packing defects from MD trajectories using grid-based
+  stamping, leaflet classification, and connected-component clustering.
+  Quantify protein-membrane interactions and contains options for visualization.
+
+
+keywords:
+  - MDAnalysis
+  - membrane
+  - protein
+  - packing defects
+  - lipids
+  - molecular dynamics
+
+license: GPL-2.0-only
+
+project_home: https://github.com/jaybraunjr/packing_defect/
```

Our scripts are not super-robust: cannot have a trailing /
```suggestion
project_home: https://github.com/jaybraunjr/packing_defect
```